Mathlib/Algebra/Module/Submodule/Localization.lean

```diff
@@ -22,7 +22,6 @@ Results about localizations of submodules and quotient modules are provided in t
 
 ## TODO
 - Statements regarding the exactness of localization.
-- Connection with flatness.
```

Comment:
Can you add a comment about where to find flatness of localization (it seems likely people will start looking here).

Reply:
Thanks! Does [e9b2dcd](https://github.com/leanprover-community/mathlib4/pull/18683/commits/e9b2dcda4b84e3aeacd2fe05cbbd229b5ec81f42) look good?

Reply:
Yes, perfect. Thanks.

Reply:
Except that there is a building error :D